apps/meteor/ee/client/omnichannel/tags/AutoCompleteTagsMultiple.tsx

```diff
@@ -41,9 +54,11 @@ const AutoCompleteTagMultiple = (props) => {
 			flexShrink={0}
 			flexGrow={0}
 			placeholder={t('Select_an_option')}
-			endReached={tagsPhase === AsyncStatePhase.LOADING ? () => {} : (start) => loadMoreTags(start, Math.min(50, tagsTotal))}
+			endReached={
+				tagsPhase === AsyncStatePhase.LOADING ? () => undefined : (start) => start && loadMoreTags(start, Math.min(50, tagsTotal))
```

Comment:
We have a lot of places doing this same pagination without the `start &&` check. Why is it necessary here?